Resources/Prototypes/_NF/Entities/Objects/Devices/flatpacks.yml

```
@@ -413,7 +413,16 @@
   - type: Flatpack
     entity: ComputerWithdrawBankATM
 
-# Medical
+- type: entity
+  parent: UniformPrinterFlatpack
+  id: ShuttleGunMailLauncherFlatpack
+  name: stationary mail launcher flatpack
+  description: A flatpack used for constructing a stationary mail launcher.
```

```suggestion
  name: FWD-6500m "Priority" flatpack
  description: A flatpack used for constructing a FWD-6500m "Priority" stationary mail launcher.
```
Whatever name you settle on, this should match.